operator/src/main/resources/scripts/encryption_util.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2019, 2020, Oracle Corporation and/or its affiliates.
```

Comment:
Consider renaming to 'model_encryption_util.py' so that it sorts next to the other model scripts.

Reply:
@jshum2479 ?

Reply:
Leaving it for later

Reply:
Please make the change or file a JIRA to track and link it here.

Reply:
I've filed a JIRA to track.  Resolving.